src/compo/modis_aod2ioda.py

```diff
@@ -123,7 +145,8 @@ def _read(self):
             sen_zen = sen_zen[pos_index]
             unc_land = unc_land[pos_index] * 1E-3  # see scale factor
```

Comment:
are we sure that those scale factors for AOD and land uncertainty are what we want really? Don't use a model unit convert factor down the line in UFO because of this and also what was hardcoded in CRTM. 
What we need to have right now is to have at least all the AOD products out of iodaconverter on the same unit range. Better check now.

Reply:
The scale factor is saved as attribute information for both AOD and uncertainty variables in the hdf4 file.

Reply:
ok but I would like us to check the consistency of this against VIIRS and possibly other AOD products